docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -1,43 +1,23 @@
 [id="using-new-subnet-ranges_{context}"]
 
-= Scenario 1: Using new subnet ranges
+= Configuring new subnet ranges
 
-This scenario is compatible with any existing subnet configuration, and can be
-used even when the existing cluster subnet ranges don't have enough free IP
-addresses for the new control plane services.
+You can define new IP ranges for control plane services that belong to a different subnet that is not used in the existing cluster. Then you configure link local IP routing between the old and new subnets to enable old and new service deployments to communicate. This involves using the {OpenStackPreviousInstaller} mechanism on a pre-adopted cluster to configure additional link local routes. This enables the EDP deployment to reach out to adopted nodes by using the old subnet addresses. You can use new subnet ranges with any existing subnet configuration, and when the existing cluster subnet ranges do not have enough free IP addresses for the new control plane services.
 
-The general idea here is to define new IP ranges for control plane services
-that belong to a different subnet that was not used in the existing cluster.
-Then, configure link local IP routing between the old and new subnets to allow
-old and new service deployments to communicate. This involves using {OpenStackPreviousInstaller}
-mechanism on pre-adopted cluster to configure additional link local routes
-there. This will allow EDP deployment to reach out to adopted nodes using their
-old subnet addresses.
+You must size the new subnet appropriately to accommodate the new control
+plane services. There are no specific requirements for the
+existing deployment allocation pools already consumed.
```

Comment:
I’m a little bit confused about what “consumed” means in this context. Already consumed by what? The new RHOSO deployment?

Reply:
Already consumed by the RHOSP (pre-adoption) environment. Context: The reason why the existing pools don't have special requirements is because they will always be "enough", because any new control plane IPs will be borrowed from the new subnet ranges; while the "old"/"previous" ranges will always be large enough to fit data plane needs - because during adoption the number of IP addresses consumed by "data plane" doesn't go up.